src/popup/router/components/FungibleTokens/TokensList.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div>
+    <div v-if="filteredResults.length > 0">
+      <TokenDisplay
+        v-for="value in filteredResults"
+        :key="value.contract || value.id"
+        :tokenData="value"
+      />
+    </div>
+    <div v-else class="tokens-msg">{{ $t('pages.fungible-tokens.no-results') }}</div>
+  </div>
+</template>
+
+<script>
+import axios from 'axios';
+import { mapState, mapGetters } from 'vuex';
+import TokenDisplay from './TokenDisplay';
+import { removeDuplicates } from '../../../utils/helper';
+
+export default {
+  components: {
+    TokenDisplay,
+  },
+  props: {
+    showMyTokens: { type: Boolean },
+    searchTerm: { type: String, default: '' },
+  },
+  data: () => ({
+    tokensPublicInfo: {},
+  }),
+  async created() {
+    try {
+      const tokens = (
+        await axios.get(
+          `https://api.coingecko.com/api/v3/coins/markets?ids=aeternity&vs_currency=${this.current.currency}`,
```

Comment:
is this correct to fetch directly from coingecko here? Why not use the backend and why fetch this in a view, not a util?

Reply:
I moved the call to an util function. 
I have discussed "Aeternity" token data visualization with the design team - currently the backend could not provide the necessary fields so I am preserving the call to coingecko api. We could further extend our token meta-information and deprecate the external api calls.

Reply:
The wallet should also be able to run independently of the tipping backend and utils. I think market prices for aeternity should be fetched directly from coingecko or whatever else provider we include instead of fetching them via the backend in that regard.